### ISSUE TITLE
Question 7, Chapter 1 

### ISSUE THREAD

**Issue by lmy931** [PacktPublishing/AWS-Certified-Machine-Learning-Specialty-MLS-C01-Certification-Guide#1]
Hi all,

I really like this book. Each chapter provides many questions and answers, which is pretty good.
but I have a concern about question 7 in Chapter 1.

The given answer is option B, but I think option C is correct as well. I will appreciate it if you could provide some explanations for option C.

You have created a classification model to predict whether a banking transaction is fraud or not. During the modeling phase, your model was performing very well on both the training and testing sets. When you executed the model in a production environment, people started to complain about the low accuracy of the model. Assuming that there was no overfitting/underfitting problem during the training phase, what would be your first line of investigation?

b) The training set used to create this model does not represent the real environment where the model was deployed.

c) The algorithm used in the final solution could not generalize enough to identify fraud cases in production.

**Comment by gaurav-packt** [comment]
Hello @lmy931,

Here is the feedback shared by the author:

*Option C it is not a root cause, but instead, it just confirms the problem. Additionally, the question states that there was no issue during the modelling phase, which kind of suggests that the algorithm was able to generalize at that time.*

*Option B then would be our best guess and the potential first line of investigation.*

*Look, option C it is not completely wrong, It depends If the tests performed during modelling phase were consistent. If they were not, the algorithm could eventually be the problem.*
*But again, the arguments that I have used before puts my first guess on option C.*

*This type of question, where multiple answers might look correct is very expected during the exam. A specialist tip here is: read carefully any hints in the question and try to understand the context of the problem.*

Let us know if this solves your query and we can close the issue.

**Comment by lmy931** [comment]
Thanks for your reply. 
The explanation is comprehensive and easy to understand. I will keep learning from this book.
I will close this issue because the question has been answered.
